src/main/java/seedu/realodex/logic/parser/ParserUtil.java

```diff
@@ -307,6 +309,21 @@ public static Birthday parseBirthday(String birthday) throws ParseException {
         return new Birthday(birthday);
     }
 
+    /**
+     * Parses a {@code String month} into a {@code Date} month.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     */
+    public static Date parseMonth(String monthName) throws ParseException {
+        assert(!monthName.isBlank() || !monthName.isEmpty());
```

Comment:
```suggestion
```

If not `filter b/` prints nothing, we cannot safely assert string here is blank

Reply:
can try again! 

`filter b/` has expected output:
Birth Month should be in MMM format.
Example: b/Sep

Reply:
🥇

Reply:
I dont think the fixes are pushed